src/app/rundown-view/components/rundown-header/rundown-header.component.scss

```diff
@@ -17,6 +17,11 @@
     background-color: var(--rundown-header-background-active-color);
   }
 
+  &--rehearsal {
+    --scrollbar-thumb-color: var(--rehearsal-color);
+    background-color: var(--rehearsal-color);
```

Comment:
Making the scrollbar thumb the same color as the background means that the user cannot see the scrollbar thumb. The scrollbar thumb should have a different color than scrollbar track.

Reply:
Fix